test/test_torch.py

```diff
@@ -10572,57 +10574,59 @@ def test_cumprod(self, device):
         # Check that output maintained correct shape
         self.assertEqual(raw_tensor.shape, raw_tensor.grad.shape)
 
-    def test_cummax(self, device):
-        x = torch.rand(100, 100, device=device)
-        out1 = torch.cummax(x, 1)
-        res2 = torch.Tensor().to(device)
-        indices2 = torch.LongTensor().to(device)
-        torch.cummax(x, 1, out=(res2, indices2))
-        self.assertEqual(out1[0], res2)
-        self.assertEqual(out1[1], indices2)
-
-        a = torch.tensor([[True, False, True],
-                         [False, False, False],
-                         [True, True, True]], dtype=torch.bool, device=device)
-        b = a.byte()
-        aRes = torch.cummax(a, 0)
-        bRes = torch.cummax(b, 0)
-        self.assertEqual(aRes[0], bRes[0])
-        self.assertEqual(aRes[0], torch.tensor([[1, 0, 1],
```

Comment:
Could you add back this test?
We should have at least one toy example that checks the output of the function.
You can have `a` fixed before and another argument to this function being the expected result here?